Tests/Marketplace/approved_usecases.json

```diff
@@ -0,0 +1,27 @@
+[
+	"Access",
+	"Breach Notification",
+	"Breach and Attack Simulation",
+	"Brute Force",
+	"Case Management",
+	"Compliance",
+	"Crisis Management",
+	"Default",
```

Comment:
I think we can remove this

Reply:
Added on purpose.  I wanted it to be easily found.